volumes/miovision/sql/function/function-clear-volumes_15min.sql

```diff
@@ -0,0 +1,45 @@
+CREATE OR REPLACE FUNCTION miovision_api.clear_volumes_15min(
```

Comment:
Can you remind me why we are not including the per intersection deleting thingy for 15_min as well?

Reply:
For faster functions I think I chose to just delete and re-insert all intersections to reduce unnecessary duplicate functions. This one is aggregating from an aggregate table (`volumes_15min_mvt`) so it's pretty fast.

Reply:
I think it would be better to keep features consistent :polite_raccoon: